src/AppBundle/Service/ReportService.php

```diff
@@ -109,7 +111,6 @@ public function createNextYearReport(Report $oldReport)
         }
 
         $this->_em->persist($newReport);
-        $this->_em->flush();
```

Comment:
removing this will not persist the newReport unless a flush is called elsewhere. Risky.